src/features/patient/__tests__/reducers.test.js

```diff
@@ -39,15 +39,17 @@ describe('patient set reducer', () => {
     expect(patientsReducer(undefined, {})).toEqual( {
```

Comment:
I'm not reading all this, let me know if there's anything I should look at...

Reply:
this was one thing I went back and forth about (and probably should be documented somewhere beyond a comment on a test case in the middle of a file):

https://github.com/openmrs/openmrs-react-components/pull/234/files#diff-92b696be9aefe0eeef32fb515a0431a7R185

Reply:
Sorry, I was just referring to the test file! I did read the rest. `isError` seems like a fine function, as far as I can tell (given my very limited Redux exposure)